universal_notifications/backends/sms/engines/twilio.py

```diff
@@ -87,8 +87,9 @@ def parse_received(self, raw):
                 #     send_email('voice_mail', user.email, 'Patient leaved a voice mail', variables)
                 pass
         elif raw.data.get('SmsStatus') == 'received':
-            receiver, c = PhoneReceiver.objects.get_or_create(number=raw.data.get('From'),
-                                                              service_number=raw.data.get('To'))
+            receiver, c = PhoneReceiver.objects.update_or_create(number=raw.data.get('From'),
+                                                              defaults={"service_number": raw.data.get('To')})
+            created = c
```

Comment:
created = c? Could we just do `receiver, created =....`?

Reply:
Sure, that is debug session leftover (guess what `c` stands for in a pdb :D)